engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -950,7 +977,11 @@ private void execute() throws YarnException, IOException
 
       if (!blacklistAdditions.isEmpty()) {
         amRmClient.updateBlacklist(blacklistAdditions, null);
-        blacklistedNodesQueueWithTimeStamp.add(new Pair<Long, List<String>>(System.currentTimeMillis(), blacklistAdditions));
+        long timeStamp = System.currentTimeMillis();
+        for(String hostname : blacklistAdditions) {
```

Comment:
Addressed the checkstyle errors. Thus didn't get caught in checkstyle because max allowed violation count was greater. Have reduced it now.